src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf

```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="OptionExplicitLabel.Text">
         <source>Option e&amp;xplicit:</source>
-        <target state="translated">Option explicit(&amp;X):</target>
+        <target state="translated">显式编译选项(&amp;X):</target>
```

Comment:
I'm finding these translations not having clear meanings. How can I send really effective feedback about translations?

Reply:
@tmeschter?

Reply:
@huoyaoyuan You can provide feedback here and we'll make sure it gets passed along (FYI, @v-zbsail, @janbaltus).

However, the best way to provide feedback on translations is via the Help | Send Feedback | Report a Problem menu item in Visual Studio.

Reply:
OK, I'll consider to open a Developer Community ticket for many translation items I found.